src/app/onion/learning-object-builder/components/scaffold/scaffold.component.scss

```diff
@@ -1,28 +1,113 @@
 @import "~_vars.scss";
 
-.text {
-  color: #646464;
-  text-align: left;
-  padding: 30px 30px;
-  &.title {
-    font-size: 16px;
+.header {
+  display: flex;
+  padding-top: 18px;
+  padding-left: 20px;  
+
+  .icon {
+    color: #4290ea;
+    font-size: 20px; 
+    align-items: left;
+    justify-content: left;
+  }
+  .title {
+    color: #646464;
+    text-align: left;
+    font-size: 20px; 
+    font-weight: 600;
+    margin: 0 0 10px 10px; 
+    align-items: left;
+    justify-content: left;
   }
-  &#placeholder {
-    font-size: 14px;
-    padding: 0px 30px 30px 30px;  }
-}
 
-.icon {
-  color: #4290ea;
-  font-size: 40px;
+  .edit{
+    color: #4290ea;
+    background-color: white; 
+    border: none; 
+    font-size: 20px;
+    margin-left: auto;
+    padding-right: 20px;
+    padding-left: 20px;
+    cursor: pointer; 
+  }
+  .edit:focus{
+    outline: 0; 
+  }
+  .edit:hover{
+    opacity: 0.6; 
+  }
 }
-div {
-  &.icon {
-    justify-content: center;
-    display: flex;
-    flex-direction: row;
-    justify-content: center;
-    align-items: center;
-    padding-top: 50px;
+
+.child-box {
+  padding: 20px 10px;
+  border-bottom: solid 1px#ccc;
+  color: rgba(0, 0, 0, 0.87);
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+  background: white;
+
+  .info {
+    float: left; 
+    .top-row {
+      display: flex; 
+      .child-title {
+        color: #646464;
+        text-align: left;
+        font-size: 16px; 
+      }
+    }
+    .bottom-row{
+      display: flex; 
+      .child-length {
+        color: #646464;
+        text-align: left;
+        font-size: 14px; 
+      }
+      .child-last-updated {
+        color: #646464; 
```

Comment:
Why is this color hard coded? Can we use one of the defined SCSS variables?

Reply:
Resolved with Add style commit